features/admin.groups.v1/pages/groups.tsx

```diff
@@ -332,36 +298,6 @@ const GroupsPage: FunctionComponent<any> = (): ReactElement => {
                         imageSize="tiny"
                     />) :
                     (<GroupList
-                        advancedSearch={ (
```

Comment:
Just to clarify, with this improvement, are we removing the filtering capability in groups page? if yes, may i ask why?

Reply:
This is a redundant search. We can see that the advancedSearch is duplicated in the list layout as well. By removing this, it will have no affect

Reply:
https://github.com/wso2/identity-apps/pull/7956/files#diff-5f68b0cff6ab31582799fbb8bdbe414ddcabfa08402bd725ce520c80c626995fL277